testing/wicket-arquillian/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<arquillian xmlns="http://jboss.org/schema/arquillian"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+    <defaultProtocol type="Servlet 3.0" />
+    <engine>
+        <property name="deploymentExportPath">target</property>
+    </engine>
+    
+    <container qualifier="wildfly" default="true">
+    	<configuration>
+    		<property name="jbossHome">target/wildfly-8.1.0.Final</property>
+            <!-- JDK before 8: -XX:MaxPermSize=128m -XX -->
+            <!-- JDK 8 and later: -XX:MaxMetaspaceSize=128m -->
+            <!-- DEBUG MODE MANAGED: -Xrunjdwp:transport=dt_socket,address=8787,server=y,suspend=y -->
+    		<property name="javaVmArguments">-Xmx128m -XX:MaxPermSize=128m -Xrunjdwp:transport=dt_socket,address=8787,server=y,suspend=n
```

Comment:
Wicket 7.x is built with JDK 7, so we have to change this before merging to master branch.